### DISCUSSION TITLE
Undo/redo on cjk content

### DISCUSSION THREAD

**Discussion by BlmVay** [PrismJS/prism#3431] [Q&A]
Because I need to enter some cjk text, but the current undo/redo function is not very compatible, how can I disable the default undo/redo?and I need to control the undo/redo stack myself.

**Comment by RunDevelopment** [comment]
Is this about [Prism live](https://github.com/PrismJS/live)? If so, then I'll transfer this discussion over.

**Comment by BlmVay** [comment]
First of all thanks for your reply, but I am using prism and the project address is https://github.com/PrismJS/prism


I am using it in a vue project.





------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Michael ***@***.***&gt;; 
发送时间: 2022年4月12日(星期二) 凌晨2:23
收件人: ***@***.***&gt;; 
抄送: "🔰BL. ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PrismJS/prism] Undo/redo on cjk content (Discussion #3431)





 
Is this about Prism live? If so, then I'll transfer this discussion over.

**Reply by RunDevelopment** [reply]
In that case, we can't help you. Prism is a syntax highlighter, not an editor. We have nothing to do with text editing, so we also don't influence the undo/redo operations of your browser or the library your project uses.

**Reply by BlmVay** [reply]
So I should try to use prism-live, right? Maybe it will solve my problem.
my english is not very good so
----------------- from Google Translate

**Reply by RunDevelopment** [reply]
Prism live is still very experimental, so it's probably not the right tool. Prism wasn't built to be used in editors, so you should probably use an editor library (e.g. monaco editor).

**Comment by BlmVay** [comment]
Thanks for your suggestion, I will try.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Michael ***@***.***&gt;; 
发送时间: 2022年4月12日(星期二) 凌晨3:32
收件人: ***@***.***&gt;; 
抄送: "🔰BL. ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PrismJS/prism] Undo/redo on cjk content (Discussion #3431)





 
Prism live is still very experimental, so it's probably not the right tool. Prism wasn't built to be used in editors, so you should probably use an editor library (e.g. monaco editor).
 
&mdash;
